core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -106,6 +129,54 @@ public void testImportMutationsFile() throws Exception {
         ImportProfileData secondRunner = new ImportProfileData(secondArgs);
         secondRunner.run();
         validateMutationAminoAcid (geneticProfileId, secondSampleId, 2842, "L113P");
+        // assume a MUTATION_COUNT record has been added for the sample and the
+        // count is 1, the other one is a germline mutation
+        // also confirm mutation count for first sample is still correct
+        clinicalData = DaoClinicalData.getSampleData(study.getInternalId(), new ArrayList<String>(Arrays.asList("TCGA-AA-3664-01", "TCGA-AA-3665-01")), clinicalAttribute);
+        assert(clinicalData.size() == 2);
+        assertEquals("8", clinicalData.get(0).getAttrVal());
+        assertEquals("1", clinicalData.get(1).getAttrVal());
+    }
+
+    @Test
+    public void testImportSplitMutationsFile() throws Exception {
+        /* Mutations file split over two files with same stable id */
```

Comment:
the test above tests a lot of things, this makes it a little cleaner by just testing the counting in the split mutation files. We can add some fusion data related tests in the PR that addresses the issue with counting fusion and germline only samples